Sources/Private/Experimental/Animations/CAAnimation+TimingConfiguration.swift

```diff
@@ -0,0 +1,73 @@
+// Created by Cal Stephens on 1/6/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import QuartzCore
+
+// MARK: - LayerAnimationContext
+
+// Context describing the timing parameters of the current animation
+struct LayerAnimationContext {
+  /// The animation being played
+  let animation: Animation
+
+  /// The timing configuration that should be applied to `CAAnimation`s
+  let timingConfiguration: ExperimentalAnimationLayer.CAMediaTimingConfiguration
+
+  /// The absolute frame number that this animation begins at
+  let startFrame: AnimationFrameTime
+
+  /// The absolute frame number that this animation ends at
+  let endFrame: AnimationFrameTime
+}
+
+// MARK: - CAAnimation + LayerAnimationContext
+
+extension CAAnimation {
```

Comment:
Moved this from `Experimental/Layers/AnimationLayer.swift` to `Experimental/Animations/CAAnimation+TimingConfiguration.swift` -- otherwise no changes